includes/collection/class-replies.php

```diff
@@ -171,10 +173,19 @@ public static function get_context_collection( $post_id ) {
 		$post_uri = ( new PostTransformer( $post ) )->to_id();
 		\array_unshift( $ids, $post_uri );
 
+		$author = Actors::get_by_id( $post->post_author );
+		if ( is_wp_error( $author ) ) {
+			if ( is_user_type_disabled( 'blog' ) ) {
+				return false;
+			}
+
+			$author = new Blog();
+		}
+
 		return array(
 			'type'         => 'OrderedCollection',
 			'url'          => \get_permalink( $post_id ),
-			'attributedTo' => Actors::get_by_id( $post->post_author )->get_id(),
```

Comment:
Maybe we need a function that always return the right Actor?

Reply:
Yes ideally we handle those kinds of fallback inside `get_by_id` so that other areas of the code don't run into similar trouble

Reply:
I am not sure that changing the `get_by_id` method is a good idea, because it might have side effects.